vsintegration/src/FSharp.VS.FSI/sessions.fs

```diff
@@ -360,6 +361,49 @@ type FsiSession(sourceFile: string) =
     /// interrupt timeout in miliseconds
     let interruptTimeoutMS = 1000
 
+    let splitPidFile () =
+        let getTfmNumber (v: string) =
+            let arr = v.Split([| '.' |], 3)
+            match Single.TryParse(arr[0] + "." + arr[1]) with
+            | true, value -> Some value
+            | _ -> None
+
+        let frameworkVersion (frameworkDescription: string) =
```

Comment:
This might deserve some unit tests...

Reply:
So that others do not have to search for it:

`The property returns a string that indicates the name of the currently executing .NET implementation. The string starts with one of the following descriptions, usually followed by the current version:

".NET" (for .NET 5 and later versions)
".NET Core" (for .NET Core 1.0 - 3.1)
".NET Framework"
".NET Native"`

Reply:
damn i forgot the comment.

Reply:
Added:
````
    // system.runtime.interopservices.runtimeinformation.frameworkdescription
    // https://learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.runtimeinformation.frameworkdescription?view=net-7.0

````